apps/web/src/hooks/useQueryParams.ts

```diff
@@ -6,10 +6,12 @@ import { Rollup } from "@blobscan/api/enums";
 import type { Rollup as LowercaseRollup } from "~/types";
 
 type QueryParams = {
+  from?: string;
   p: number;
   ps: number;
   rollup?: LowercaseRollup | "null";
```

Comment:
I think the `rollup` shouldn´t be a query param as we use the `from` mapped address associated instead

Reply:
Actually, we still need it in order to filter non-rollup elements (`rollup=null`)